### PR TITLE
Cloud service autodetection feature documentation 

### PR DESCRIPTION
**Pull request by kassenq**
### What does this PR do?
Adds documentation to the NPM page under "Queries" that informs readers on how to make queries for seeing traffic to AWS cloud services

### Motivation
Cloud service autodetection is a new feature in NPM that we'd like to include in the documentation.

### Preview
https://docs-staging.datadoghq.com/npm-cloud-service-autodetection/network_monitoring/performance/network_page/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/network_monitoring/performance/network_page.md [hunk 1]
```diff
@@ -38,6 +38,17 @@ You can set the timeframe over which traffic is aggregated using the time select
...
 
+### Cloud service autodetection
+Filtering by specific third-party services such as S3, RDS, Kinesis, and ELB can help pinpoint latency, assess database performance, and visualize your network more completely. For instance, you can filter a search query by service, view the service in the Network Map, and trace communication on that node to see affected services.
+
+{{< img src="network_performance_monitoring/network_page/cloud-service-hero-docs.png" alt="Cloud Service Map" >}}
+
+- To filter a query: in search bar(s), enter tags such as `service:s3`, `service:kinesis`, and `service:elb`. For a number of services, you can break down latency and retransmits further by using more out-of-the-box tags like `s3_bucket` and `rds_instance_type`.
+- To visualize specific managed services: in the [Network Map][1], click the dropdown next to *View* and type or select desired tags. In the map, click a particular node to view troubleshooting options.
+- To view integration metrics for a service: in the Network Page, click on a row in the [dependency table](#table). In the opened side panel, click *Integration Metrics*.
```
Comment (shlomoartzi) [inline content/en/network_monitoring/performance/network_page.md:48]: the last point on integration metrics can focus more on value - e.g. use the integration metrics tab to analyze the performance of the cloud service and distinguish b/w a client-side and cloud provider-side issue

File: content/en/network_monitoring/performance/network_page.md [hunk 2]
```diff
@@ -107,6 +107,17 @@ TCP is a connection-oriented protocol that guarantees in-order delivery of packe
 | **Established Connections** | The number of TCP connections in an established state. Measured in connections per second from the `source`. |
 | **Closed Connections** | The number of TCP connections in a closed state. Measured in connections per second from the `source`. |
 
+### Cloud service autodetection
+Filtering by specific third-party services such as S3, RDS, Kinesis, and ELB can help pinpoint latency, assess database performance, and visualize your network more completely. For instance, you can filter a search query by service, view the service in the Network Map, and trace communication on that node to see affected services.
```
Comment (kayayarai) [inline content/en/network_monitoring/performance/network_page.md:111]:
```suggestion
By filtering on third-party services such as S3, RDS, Kinesis, and ELB, you can pinpoint latency, assess database performance, and visualize your network more completely. For instance, you can filter a search query by service, view the service in the Network Map, and trace communication on that node to see affected services.
```

File: content/en/network_monitoring/performance/network_page.md [hunk 3]
```diff
@@ -107,6 +107,17 @@ TCP is a connection-oriented protocol that guarantees in-order delivery of packe
...
+
+{{< img src="network_performance_monitoring/network_page/cloud-service-hero-docs.png" alt="Cloud Service Map" >}}
+
+- To filter a query: in search bar(s), enter tags such as `service:s3`, `service:kinesis`, and `service:elb`. For a number of services, you can break down latency and retransmits further by using more out-of-the-box tags like `s3_bucket` and `rds_instance_type`.
```
Comment (kayayarai) [inline content/en/network_monitoring/performance/network_page.md:115]:
```suggestion
- **To filter a query**: In a search bar, enter tags such as `service:s3`, `service:kinesis`, and `service:elb`. For some services, you can break down latency and retransmits further by using more out-of-the-box tags like `s3_bucket` and `rds_instance_type`.
```

File: content/en/network_monitoring/performance/network_page.md [hunk 4]
```diff
@@ -107,6 +107,17 @@ TCP is a connection-oriented protocol that guarantees in-order delivery of packe
...
+- To visualize specific managed services: in the [Network Map][2], click the dropdown next to *View* and type or select desired tags. In the map, click a particular node to view troubleshooting options.
```
Comment (kayayarai) [inline content/en/network_monitoring/performance/network_page.md:116]:
```suggestion
- **To visualize specific managed services**: In the [Network Map][2], click the dropdown next to *View* and type or select desired tags. In the map, click a node to view troubleshooting options.
```

File: content/en/network_monitoring/performance/network_page.md [hunk 5]
```diff
@@ -107,6 +107,17 @@ TCP is a connection-oriented protocol that guarantees in-order delivery of packe
...
+- To view integration metrics for a service: in the Network Page, click on a row in the [dependency table](#table). In the opened side panel, use the *Integration Metrics* tab to analyze the performance of cloud services and distinguish between a client-side and cloud provider issue.
```
Comment (kayayarai) [inline content/en/network_monitoring/performance/network_page.md:117]:
```suggestion
- **To view integration metrics for a service**: In the Network Page, click a row in the [dependency table](#table). In the opened side panel, use the *Integration Metrics* tab to analyze the performance of cloud services and distinguish between a client-side and cloud provider issue.
```

File: content/en/network_monitoring/performance/network_page.md [hunk 6]
```diff
@@ -107,6 +107,17 @@ TCP is a connection-oriented protocol that guarantees in-order delivery of packe
...
+
+To monitor other endpoints where an Agent cannot be installed (such as public APIs), group the destination in the Network Overview by  the [`domain` tag](#dns-resolution).
```
Comment (kayayarai) [inline content/en/network_monitoring/performance/network_page.md:119]:
```suggestion
To monitor other endpoints where an Agent cannot be installed (such as public APIs), group the destination in the Network Overview by the [`domain` tag](#dns-resolution).
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Thank you for this PR! Some minor style edits for you.

**Review by kayayarai — APPROVED** [review]

**Review by shlomoartzi — APPROVED** [review]

**Pull request merged**